components/devtools/actors/console.rs

```diff
@@ -89,6 +108,86 @@ pub struct ConsoleActor {
     pub streams: RefCell<Vec<TcpStream>>,
 }
 
+impl ConsoleActor {
+    fn evaluateJS(
```

Comment:
This function is moved as-is from the "evaluateJS" handler so that "evaluateJS" and "evaluateJSAsync" can share it